mobile/examples/speech_recognition/android/app/src/main/java/ai/onnxruntime/example/speechrecognition/TensorUtils.kt

```diff
@@ -0,0 +1,20 @@
+package ai.onnxruntime.example.speechrecognition
+
+import ai.onnxruntime.OnnxTensor
+import ai.onnxruntime.OrtEnvironment
+import java.nio.FloatBuffer
+import java.nio.IntBuffer
+
+internal fun createIntTensor(env: OrtEnvironment, data: IntArray, shape: LongArray): OnnxTensor {
+    return OnnxTensor.createTensor(env, IntBuffer.wrap(data), shape)
+}
+
+internal fun createFloatTensor(
+    env: OrtEnvironment,
+    data: FloatArray,
+    shape: LongArray
+): OnnxTensor {
+    return OnnxTensor.createTensor(env, FloatBuffer.wrap(data), shape)
+}
+
+internal fun tensorShape(vararg dims: Long) = longArrayOf(*dims)
```

Comment:
Good to know there's a `vararg` keyword in kotlin.

assuming it is safe to pass in zero elements and also only works for the same type series of arguments?

Reply:
yes. more info: https://kotlinlang.org/docs/functions.html#variable-number-of-arguments-varargs